tests/orion/services/test_flow_run_notifications.py

```diff
@@ -99,6 +100,24 @@ async def test_service_sends_notifications(
     )
 
 
+@pytest.mark.parametrize(
+    "api_url,ui_url",
+    [
+        (None, "http://ephemeral-orion/api"),
+        ("https://api.prefect.cloud/api", "https://app.prefect.cloud"),
```

Comment:
@zangell44 can you confirm this is the correct conversion?

Reply:
Yep this looks good!